prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -247,6 +252,11 @@ open class Receiver(
                 }
             }
         }
+
+        if (topic != Topic.FULL_ELR && conditionFilter.isNotEmpty() && conditionFilter[0].isNotEmpty()) {
+            return "Invalid value for condition filter. " +
+                "Cannot have a condition filter in pipelines other than universal pipeline."
```

Comment:
`Invalid value` seems to me to be a little bit misleading, the value isn't necessarily invalid, it's more that using the field at all is prohibited. I might prefer a message like `Condition filter only allowed for receivers with topic "full_elr", but one was found for receiver $fullName`. That's not great either, so consider this optional, what you have is probably fine.